lldb/tools/lldb-dap/Protocol/ProtocolTypes.cpp

```diff
@@ -43,6 +43,32 @@ bool fromJSON(const json::Value &Params, Source &S, json::Path P) {
          O.map("sourceReference", S.sourceReference);
 }
 
+static llvm::json::Value ToString(PresentationHint hint) {
```

Comment:
toJSON?

Then on line 67 you don't need to add `ToString`, it would be handled by the existing overloads.

Reply:
:100: